---
title: Choriocapillaris participate in the progress of age-related macular degeneration
  by regulating COL10A1 expressed by retina pigment epithelial
author: "Chunyou Yin^1\\*^; Chunhui Yin^2^"
csl: elsevier-with-titles-alphabetical.csl
output:
  bookdown::word_document2: 
    fig_caption: yes
    number_sections: false
    link-citations: yes

bibliography:
- /home/yincy/git/R/rmarkdown/library.bib
- /home/yincy/git/Data/mendeley.bib/library.bib
- /home/yincy/git/R/rmarkdown/-mendeley.bib
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = F, 
                      messages = F, 
                      warning = F,
                      fig.width = 8, 
                      fig.height = 9,
                      fig.align = "center", 
                      dpi = 300, 
                      message = F, 
                      warning = F)
```

```{r load-required-packages, message=FALSE, warning=FALSE, echo=FALSE}
library(tidyverse)
library(monocle3)
library(nichenetr)
library(cowplot)
```

```{r load-single-cell-image}
cc <- readRDS(file = "/home/yincy/git/R/Retina/res/cc.rds")
expression_ligands_ridges <- readRDS(file = "../Retina/res/expression_ligands_ridges.rds") +
  scale_y_discrete(name = "top ligands")
```

```{r single-cell-plot, message=FALSE, warning=FALSE}
pseudotime_p <- plot_cells(cds = cc, 
           color_cells_by = "pseudotime", 
           show_trajectory_graph = T, 
           cell_size = 1.2, 
           trajectory_graph_segment_size = 1, 
           label_leaves = F, 
           label_roots = F)

percent.mt_p <- plot_cells(cds = cc, 
                           color_cells_by = "percent.mt", 
                           cell_size = 1.2, 
                           trajectory_graph_segment_size = 1, 
                           show_trajectory_graph = T, 
                           label_roots = F, 
                           label_leaves = F)

expression_of_TGFB1 <- plot_cells(cds = cc, 
                                  reduction_method = "UMAP", 
                                  genes = "TGFB1", 
                                  show_trajectory_graph = T, 
                                  trajectory_graph_segment_size = 0.2, 
                                  cell_size = 1, 
                                  min_expr = 0, 
                                  label_leaves = F, 
                                  label_roots = F) +
  scale_color_gradient(name = "relative\nexpression", low = "green", high = "red")

expression_of_COL18A1 <- plot_cells(cds = cc, 
                                  reduction_method = "UMAP", 
                                  genes = "COL18A1", 
                                  show_trajectory_graph = T, 
                                  trajectory_graph_segment_size = 0.2, 
                                  cell_size = 1, 
                                  min_expr = 0, 
                                  label_leaves = F, 
                                  label_roots = F) +
  scale_color_gradient(name = "relative\nexpression", low = "green", high = "red")
```


# Supplementary Materials  
### differential expression of ligands along the trajectory  
```{r statistic-result-ligands, fig.cap="The statistical significance of ligands expressed along the trajectory in the choriocapillaris endothelial cells."}
cc_graph_test_res <- read.csv(file = "/home/yincy/git/R/Retina/res/cc_graph_test_res.csv")

top_ligands <- c("IL1B", "TGFB1", "ADAM17", "IL33", 
                 "CXCL12", "CXCL2", "COL18A1", "CXCL3", 
                 "IHH", "OCLN", "HMGB1", "FGF2", 
                 "APP", "HSP90B1", "BMP2")

o <- cc_graph_test_res %>% 
  dplyr::filter(gene_short_name %in% top_ligands) %>% 
  arrange(q_value) %>% 
  pull(gene_short_name)

cols <- rep("grey30", length(o))
names(cols) <- o
cols["TGFB1"] <- "blue"

cc_graph_test_res %>% 
  dplyr::filter(gene_short_name %in% top_ligands) %>% 
  ggplot(aes(reorder(gene_short_name, q_value), -log10(q_value))) +
  geom_bar(stat = "identity", fill = cols) +
  scale_y_continuous(name = "-log10(q_value)", breaks = seq(0, 50, 5)) +
  scale_x_discrete(name = NULL) +
  coord_flip() +
  theme_classic()
```


### differential expression of targets  
```{r statistic-result-targets, fig.cap="The expression of targets expressed in RPE, only COL10A1 and TLR4 are significantly differentially expressed among health(n = 3) and early AMD patients(n = 7)."}
statistic_significance_targets <- readRDS(file = "/home/yincy/git/R/Retina/res/statistic_significance_targets.rds")

plot_grid(statistic_significance_targets + theme_classic(), 
          nrow = 2, 
          rel_heights = rep(1, 2)) 
```


### differential expression of COL18A1 along the trajectory
```{r, fig.cap="The expression of COL18A1 was elevated in the progress of AMD in the choriocapillaris endothelial cells."}
plot_grid(expression_of_COL18A1, 
          ncol = 2, 
          rel_widths = c(3, 1))
```


### differential expression of regulators  
```{r statistic-targets, message=FALSE, warning=FALSE, fig.cap="There are 4 transcription regulators that can regulator COL10A1's expression, after filtering only RUNX2, SMAD3, and ESR1 are expressed, and only RUNX2 significantly down regulated in early AMD pathients(n = 3) compare to the health(n = 7)."}
statistic_significance_regulator <- readRDS(file = "/home/yincy/git/R/Retina/res/statistic_significance_regulator.rds")
plot_grid(statistic_significance_regulator, 
          nrow = 2)
```


### dimensional reduction of bulk-seq health and early AMD patients  
```{r, fig.cap="Dimensional reduction of bulk-seq health(n = 7) and early AMD patients(n = 3), which show the high quality of the sequencing data, as health and early AMD patients can be clearly separated."}
knitr::include_graphics(path = "../Retina/res/dimreduc_bulk_normal_amd.tiff")
```


### Software and R/Bioconductor packages  
Name               | website 
-------------------|---------------------------------------------------------------
R                  | https://www.r-project.org
RStudio            | https://rstudio.com/
edgeR              | http://www.bioconductor.org/packages/release/bioc/html/edgeR.html
limma              | http://www.bioconductor.org/packages/release/bioc/html/limma.html
monocle3           | http://cole-trapnell-lab.github.io/monocle-release/
tidyverse          | https://www.tidyverse.org/
cowplot            | https://github.com/wilkelab/cowplot
RColorBrewer       | https://CRAN.R-project.org/package=RColorBrewer
nichenetr          | https://github.com/saeyslab/nichenetr
GEOquery           | http://www.bioconductor.org/packages/release/bioc/html/GEOquery.html
ggridges           | https://CRAN.R-project.org/package=ggridges
EnsDb.Hsapiens.v86 | http://www.bioconductor.org/packages/release/data/annotation/html/EnsDb.Hsapiens.v86.html
knitr              | https://yihui.org/knitr/
rmarkdown          | https://github.com/rstudio/rmarkdown
bookdown           | https://github.com/rstudio/bookdown


### sessionInfo
```{r sessionInfo}
sessionInfo <- readRDS(file = "/home/yincy/git/R/Retina/res/sessionInfo.rds")
sessionInfo
```
